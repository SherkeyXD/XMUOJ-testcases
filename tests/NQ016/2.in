-1EADBBF4 -176E2A60
-39F144C7 -1995E2A8
-23F5CB94 -2123D872
-38039044 -2F19A5F5
-2895B433 -3B5B9268
-15B933AC -36B95D86
-30E57A5A -3A7930B1
-3B45A084 -39E976CE
-312A5AF8 -3A888639
-3A9A4C01 -39960AF9
-1284E53C -275A99DA
-3B5B2779 -3892825C
-158F1C8A -2B359380
-3B74C594 -3329B858
-1961D334 -30171364
-3AEFBB0A -3340541E
-396C5D2A -397F33AE
-37675378 -210D8CC4
-369F7DC2 -2EFD4F52
-348C0100 -39E40B40
-293F0F08 -1C0D6509
-3735A624 -270277BD
-28CC056F -30AC4998
-31787680 -39E25874
-39EB333C -334DEDE6
-31AC3C48 -3A2F0404
-35CC02CF -2092AAC0
-2F2AF188 -1753AF80
-1782BF60 -2788DC83
-3A51D100 -15177D1
-3422CA38 -2500E2B0
-38B569D -2CF617C0
-1762BE3F -50AD9D0
-36FF1915 -1C045AB0
-32E6D728 -349EE625
-2CC146E4 -366A7DA4
-3B6C3848 -2314D1C9
-25F7B9CC -3A8A6280
-25CD8CA8 -30AE7DDD
-3B8AABA2 -373D9FAA
-2959E138 -32016F1C
-135AA4D0 -3427B418
-2A468D0C -1ED2EBF0
-24AF6CB0 -28B41DC0
-35E14F18 -3AD64A72
-23758710 -35AC29FE
-BEC13EF -362C6040
-340C420F -23BFF3FC
-2725E460 -32A480D6
-2A4D4016 -30DEC3BA
-33C66D42 -30725B6E
-2F945215 -3A123C68
-392176AB -23980998
-2BF60C80 -ACC1208
-339D51A0 -352B85F9
-231F3B46 -22BD33BA
