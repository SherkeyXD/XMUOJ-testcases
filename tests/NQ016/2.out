-361BE654
-5387276F
-4519A406
-671D3639
-63F1469B
-4C729132
-6B5EAB0B
-752F1752
-6BB2E131
-743056FA
-39DF7F16
-73EDA9D5
-40C4B00A
-6E9E7DEC
-4978E698
-6E300F28
-72EB90D8
-5874E03C
-659CCD14
-6E700C40
-454C7411
-5E381DE1
-59784F07
-6B5ACEF4
-6D392122
-6BDB404C
-565EAD8F
-467EA108
-3F0B9BE3
-3BA348D1
-5923ACE8
-30816E5D
-1C6D980F
-530373C5
-6785BD4D
-632BC488
-5E810A11
-60821C4C
-567C0A85
-72C84B4C
-5B5B5054
-478258E8
-491978FC
-4D638A70
-70B7998A
-5921B10E
-4218742F
-57CC360B
-59CA6536
-5B2C03D0
-6438C8B0
-69A68E7D
-5CB98043
-36C21E88
-68C8D799
-45DC6F00
